css-canvas-medical/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <!-- 1st container: header [image | button x 4] -->
+  <div class="container-one">
+    <!-- button x 4 -->
+    <div id="menu-button">
+      <button type="button" onclick="location.href='product.html'" class="uncolored-button">Product</button>
+      <button type="button" onclick="location.href='implementation.html'" class="uncolored-button">Implementation</button>
+      <button type="button" onclick="" class="uncolored-button">About Us</button>
+      <button type="button" onclick="" class="uncolored-button">Blog</button>
+      <button type="button" onclick="" class="colored-button">Contact Us</button>
```

Comment:
I would replace these buttons with a `ul` and `lis` with `a` elements for the navigation. It is a better practice for navigation.